static/schemas/2.0/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "workflowGraphs.2.0",
+    "definitions": {
+        "tasks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "oneOf": [
+                    {"required": ["taskName"]},
+                    {"required": ["taskDefinition"]}
```

Comment:
Since `label` is required to each task, so add the `label` into the required list.